package-lock.json

```diff
@@ -1,8 +1,15806 @@
 {
   "name": "vscode-arduino",
   "version": "0.4.8",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
Looks like the lockfile version updated because you're using NPM v7+. There's nothing inherently wrong with that but given that this PR didn't actually touch any `package.json` files, can you exclude both `package-lock.json` files from the PR to reduce risk?